app/views/facilities/show_users.html.erb

```diff
@@ -0,0 +1,93 @@
+              Primary Nurses
+            </h3>
+          </div>
+          <ul class="divide-y divide-gray-200 h-64">
+            <% @facility.primary_nurses.each do |nurse| %>
+              <li class="text-gray-800 list-disc ml-4 text-sm flex items-center justify-between my-2 mx-8">
+                <h1><%= nurse.full_name %></h1>
+                <% if policy(@facility).unassign_facility? %>
+                  <%= form_with scope: :facility, url: unassign_facility_path, method: :put do |form| %>
+                    <%= form.hidden_field "facility_id", value: @facility.id %>
+                    <%= form.hidden_field "nurse_id", value:nurse.id %>
+                    <%= button_tag "", {
+                      class: "bg-red-50 rounded-md p-1.5 text-red-500 hover:bg-red-100 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-red-50 focus:ring-red-600",
+                      onClick: "this.form.submit()"
+                      } do %>
+                      <svg class="h-3 w-3" x-description="Heroicon name: solid/x" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
+                        <path fill-rule="evenodd" d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z" clip-rule="evenodd"></path>
+                      </svg>
+                    <% end %>
+                  <% end %>
+                <% end %>
+              </li>
+            <% end %>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="w-1/2 ">
+    <div class="max-w-7xl mx-auto py-6 sm:px-6 lg:px-8">
+      <div class="max-w-none mx-auto">
+        <div class="bg-white overflow-hidden sm:rounded-lg sm:shadow">
+          <div class="bg-white px-4 py-5 border-b border-gray-200 sm:px-6">
+            <h3 class="text-lg leading-6 font-medium text-gray-900">
+              Secondary Nurses
+            </h3>
+          </div>
+          <ul class="divide-y divide-gray-200 h-64">
+            <% @facility.secondary_nurses.map do |nurse| %>
+              <li class="text-gray-800 list-disc ml-4 text-sm flex items-center justify-between my-2 mx-8">
+                <h1><%= nurse.full_name %></h1>
+                <% if policy(@facility).unassign_facility? %>
+                  <%= form_with url: unassign_facility_path, method: :put do |form| %>
+                    <%= form.hidden_field "facility[facility_id]", value: @facility.id %>
+                    <%= form.hidden_field "facility[nurse_id]", value:nurse.id %>
+                    <%= button_tag "", {
+                      class: "bg-red-50 rounded-md p-1.5 text-red-500 hover:bg-red-100 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-red-50 focus:ring-red-600",
+                      onClick: "this.form.submit()"
+                      } do %>
```

Comment:
Instead of `button_tag`, can use `form.submit` 🤔
Have seen this in many places. Replacing all instances would be good. 🎉